code/Code/style.css

```diff
@@ -0,0 +1,151 @@
+/***ALLGEMEIN***/
+@import url("https://fonts.googleapis.com/css2?family=Abel&display=swap");
+
+body {
+  font-family: "Abel", sans-serif;
+  color: black;
+  margin: 0;
+}
+
+/***NAVIGATION***/
+
+.menu-parent {
+  border: 0px solid transparent;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  flex-direction: row;
```

Comment:
Nice use of flexbox! Did you know that `flex-direction: row;` is default for flexbox? You could save a line by removing it 😄